WallsAndHoles/propertybrowser.cpp

```diff
@@ -252,3 +258,37 @@ void PropertyBrowser::setLine(QString propertyName, QVariant value)
 
     mPropertyManager->blockSignals(false);
 }
+
+void PropertyBrowser::blankLine(QString propertyName)
+{
+    //Prevents a signal loop
+    mPropertyManager->blockSignals(true);
```

Comment:
Are we guaranteed that only the mPropertyManager will ever trigger a blankLine() call?

Reply:
(like in the future too)